pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1064,7 +1064,7 @@ type Service struct {
 
 // PDStatus is PD status
 type PDStatus struct {
-	Synced      bool                    `json:"synced,omitempty"`
+	Synced      bool                    `json:"synced"`
```

Comment:
Do we need to add comment `//+option`? because removed the `omitempty`, the field becomes required.

Reply:
Yes, it's needed.